Userland/Shell/main.cpp

```diff
@@ -171,7 +171,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     StringView format;
     bool should_format_live = false;
     bool keep_open = false;
-    bool posix_mode = false;
+    bool posix_mode = (LexicalPath::basename(arguments.strings[0]) == "sh"sv);
```

Comment:
Extra parens: intentional?

Reply:
Kind of, I dislike the less-readability of the non-parens one.